File: java/app/src/main/java/com/anderysinitsyn/myapplicationjava/MainActivity.java
package com.anderysinitsyn.myapplicationjava;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView mHelloTextView;
    private EditText mNameEditText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mHelloTextView = findViewById(R.id.textView);
        mNameEditText = findViewById(R.id.editTextTextPersonName);
    }

    public void onClick(View view) {
        if (mNameEditText.getText().length() == 0) {
            mHelloTextView.setText("Hello Kitty");
        } else {
            mHelloTextView.setText("Привет, " + mNameEditText.getText());
        }

    }

}

File: AK/example_1/app/src/main/java/com/anderysinitsyn/l1/MainActivity.kt
package com.anderysinitsyn.l1

import android.os.Bundle
import android.widget.ImageButton
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private lateinit var mHelloTextView: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mHelloTextView = findViewById(R.id.textView2)
        val imageButton: ImageButton = findViewById(R.id.imageButton2)

        imageButton.setOnClickListener {
            mHelloTextView.text = "hello Kitty"
        }
    }
}